moon/skyline.py

```diff
@@ -0,0 +1,100 @@
+"""
+  Moon Skyline analysis
+"""
+import math
+
+import numpy as np
+import cv2  # for debug
+
+
+def skyline(img):
+    b = img[:,:,0]
+    g = img[:,:,1]
+    r = img[:,:,2]
+    mask = r > 10
+    return mask.argmax(axis=0)
+
+
+def find_peaks(arr):
+    x = arr.argmin()
+    y = arr[x]
+    ret = []
+    if x > 0:
+        ret.append((x, y))
+
+        # try to remove the single peak
+        # TODO numpy optimize
+        i = x
+        while i > 0 and arr[i - 1] >= arr[i]:
+            i -= 1
+        min_i = i
+
+        i = x
+        while i + 1 < len(arr) and arr[i] <= arr[i + 1]:
+            i += 1
+        max_i = i
+
+        arr2 = arr.copy()
+        arr2[min_i:max_i+1] = arr.max()
+
+        x = arr2.argmin()
+        y = arr2[x]
+        if x > 0:
+            ret.append((x, y))
```

Comment:
Btw, your PR comment says that you are looking for a single peak.

Reply:
that was the first draft, then I was looking for the next one and comparing results from both cameras. Now I would use all available and ignore stereo (use them separately, the angle is sufficient and distance estimate is much better from lidar, if it is available)